support-frontend/webpack.common.js

```diff
@@ -95,6 +95,7 @@ module.exports = (cssFilename, jsFilename, minimizeCss) => ({
     chunkFilename: `webpack/${jsFilename}`,
     filename: `javascripts/${jsFilename}`,
     publicPath: '/assets/',
+    strictModuleExceptionHandling: process.env.NODE_ENV === 'production',
```

Comment:
Is this why the build didn't break?

Reply:
Yes, this should throw an error if an import can't be resolved, so would fail the build rather than just generate a warning.

Reply:
Yes, I was getting a warning about it... very weird that it didn't break